docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md

```diff
@@ -1,100 +1,132 @@
 
    ```xml
    <AllFormFactors>
    ...
    <Page>
-   <SourceLocation resid="ContosoAddin.Url"/>
+     <SourceLocation resid="Taskpane.Url"/>
    </Page>
    ...
    ```
 
-5. In the `<DesktopFormFactor>` section, change the **FunctionFile** from **Commands.Url** to use **ContosoAddin.Url**.
+1. Find the `<FunctionFile ...>` tag and change the `resid` from **Commands.Url** to  **Taskpane.Url**.
 
-   ```xml
-   <DesktopFormFactor>
-   <GetStarted>
-   ...
-   </GetStarted>
-   <FunctionFile resid="ContosoAddin.Url"/>
-   ```
+    ```xml
+    </GetStarted>
+    ...
+    <FunctionFile resid="Taskpane.Url"/>
+    ...
+    ```
 
-6. In the `<Action>` section, change the source location from **Taskpane.Url** to **ContosoAddin.Url**.
+1. Save the **manifest.xml** file.
 
-   ```xml
-   <Action xsi:type="ShowTaskpane">
-   <TaskpaneId>ButtonId1</TaskpaneId>
-   <SourceLocation resid="ContosoAddin.Url"/>
-   </Action>
-   ```
+## Configure the webpack.config.js file
 
-7. Add a new **Url id** for **ContosoAddin.Url** that points to **taskpane.html**.
+The **webpack.config.js** will build multiple runtime loaders. You need to modify it to load only the shared JavaScript runtime via the **taskpane.html** file.
 
-   ```xml
-   <bt:Urls>
-   <bt:Url id="Functions.Script.Url" DefaultValue="https://localhost:3000/dist/functions.js"/>
-   ...
-   <bt:Url id="ContosoAddin.Url" DefaultValue="https://localhost:3000/taskpane.html"/>
-   ...
-   ```
+1. Start Visual Studio Code and open the Excel or PowerPoint add-in project you generated.
+1. Open the **webpack.config.js** file.
+1. Remove the following **functions.html** plugin code.
+
+    ```javascript
+    new HtmlWebpackPlugin({
+        filename: "functions.html",
+        template: "./src/functions/functions.html",
+        chunks: ["polyfill", "functions"]
+      })
+    ```
+
+1. Remove the following **commands.html** plugin code.
+
+    ```javascript
+    new HtmlWebpackPlugin({
+        filename: "commands.html",
+        template: "./src/commands/commands.html",
+        chunks: ["polyfill", "commands"]
+      })
+    ```
 
-8. Save your changes and rebuild the project.
+1. Add **functions** and **commands** entries to the chunks list as shown next.
+
+    ```javascript
+      new HtmlWebpackPlugin({
+        filename: "taskpane.html",
+        template: "./src/taskpane/taskpane.html",
+        chunks: ["polyfill", "taskpane", "commands", "functions"]
+      })
+    ```
+
+1. Save your changes and rebuild the project.
 
    ```command line
    npm run build
    ```
 
+> [!NOTE]
+> You can also remove the **functions.html** and **commands.html** files. The **taskpane.html** will load the **functions.js** and **commands.js** code into the shared JavaScript runtime via the webpack updates you just made.
+
+1. Save your changes and run the project. Ensure that it loads and runs with no errors.
+
+   ```command line
+   npm start
```

Comment:
I got this error in the workbook that opens for this step:
![image](https://user-images.githubusercontent.com/38896772/133852490-e21fa28b-ca51-4a17-8017-311233800925.png)

Reply:
Okay, fixing the manifest fixed this error. Shocking, I know...